python/prairielearn.py

```diff
@@ -52,6 +54,99 @@ class QuestionData(TypedDict):
 class ElementTestData(QuestionData):
     test_type: Literal['correct', 'incorrect', 'invalid']
 
+def grade_question_parameterized(
```

Comment:
@eliotwrobson I think this is leaking from another PR?

Reply:
This is intentional. This is a shared function that elements can use to cut down on boilerplate, and this is needed to make this element work. It'll disappear from the diff once it gets into master at least once.